tests/reallylongtestnametomakesurethatthedbworksfinewithlongname.test/runit

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env bash
```

Comment:
I think an empty `runit` file is good enough to test your change. It'd be great if we could keep tests simple and easy to understand.

Reply:
We need to test client connecting to long named db.
This is a copy of basic.test ... i can put that in the readme.

Reply:
`setup` will issue `SELECT 1` to ping the database. It already tests your change in a way.

Reply:
I added a comment that runit is copied from basic.test, maybe i can in Makefile do the cp so there would be no runit in that directory.